### ISSUE TITLE
Immutable OrderedMap shows up in the wrong order

### ISSUE THREAD

**Issue by jwyuen** [zalmoxisus/redux-devtools-extension#339]
Hello!  I'm using ImmutableJS data structures with my redux store and it mostly works with Redux DevTools.  I'm using the extension downloaded from the Chrome store (v2.14.3).  One thing I've noticed is that when I use OrderedMap, the order of the items inside don't get displayed correctly when viewing the state in the DevTools.  I did verify that they are in the correct order though using console.log so I think it's an issue with DevTools....

**Comment by zalmoxisus** [comment]
Yes, the problem is that it's converted into an object to be passed to the monitors. We should convert it to ES6 Map as we got support for it recently.
